Operators/Lib/io/midi/MidiSysexOutput.cs

```diff
@@ -0,0 +1,199 @@
+
+        if (!_initialized)
+        {
+
+            if (!_registered)
+            {
+                // Register the midi device
+                MidiConnectionManager.RegisterConsumer(this);
+                _registered = true;
+            }
+
+            var contextval = SysexString.GetValue(context);
+            
+            // Split the string by space character
+            var characters = contextval.Split(' ');
+                    
+            // Iterate and build our byte list
+            foreach (string word in characters)
+            {
+                try
+                {
+                    byte byteValue = byte.Parse(word, System.Globalization.NumberStyles.HexNumber);
+                    _bytes.Add(byteValue);
+                }                
+                catch (Exception e)
+                {
+                    _lastErrorMessage = $"Failed to convert [{word}] to a byte: " + e.Message;
+                    Log.Warning(_lastErrorMessage, this);
+                    
+                    // Set initialized so we do not loop
+                    _initialized = true;
+                    return;
+                }
+            }
+            
+            // If the starting byte of the sysex string does not equal midi exclusive start
+            if (_bytes.ElementAt(0) != 240)
+            {
+                Log.Error($"Sysex String needs to start with midi exclusive start byte: F0", this);
+                
+                // Set initialized so we do not loop
+                _initialized = true;
+                return;
+            }
+                
+            // If the end byte of the sysex string does not equal midi exclusive end
+            if (_bytes.ElementAt(_bytes.Count-1) != 247)
+            {
+                Log.Error($"Sysex String needs to end with midi exclusive end byte: F7", this);
+                
+                // Set initialized so we do not loop
+                _initialized = true;
+                return;
+            }
+            
+
+            foreach (var (m, device) in MidiConnectionManager.MidiOutsWithDevices)
+            {
+                if (device.ProductName != deviceName)
+                    continue;
+
+                try
+                {
+
+                    if (_bytes != null && triggerActive)
+                    {
+                        m.SendBuffer(_bytes.ToArray());
+                        
+                        Log.Debug($"Sent Sysex: [ " + contextval + " ] To: [ " + deviceName + " ]", this);
+                    }
+
+                    foundDevice = true;
+                    break;
+
+                }
+                catch (Exception e)
+                {
+                    _lastErrorMessage = $"Failed to send midi to {deviceName}: " + e.Message;
+                    Log.Warning(_lastErrorMessage, this);
+                }
+            }
+
+            _lastErrorMessage = !foundDevice ? $"Can't find MidiDevice {deviceName}" : null;
```

Comment:
You can have a better display than error logs for those kind of blockers, cf https://github.com/tixl3d/tixl/blob/main/Operators/Lib/io/osc/OscInput.cs `SetStatus()` methods , for an example.